book.toml

```diff
@@ -43,8 +43,14 @@ exclude = [
     "objects\\.githubusercontent\\.com",
     # The bug listing URL works only if an user is logged in, otherwise one gets 404.
     "github\\.com/issues\\?",
+    # Similarly 500 is sometimes returned here.
+    "github\\.com/rust-lang/rust/pulls\\?",
```

Comment:
I'm not sure we want to skip this, isn't PR number linking kinda important

Reply:
Yeah, it is. I want to skip only the 2 particular queries that bring the troubles:
```
https://github.com/rust-lang/rust/pulls?q=is%3Apr+label%3AS-inactive+is%3Aclosed
https://github.com/rust-lang/rust/pulls?q=is%3Apr+label%3AS-inactive
```

Does it make sense?

Reply:
Oh okay, that makes sense. But also at the same time, it's horrifying(?) that these endpoints can return internal server errors?

Reply:
@marxin out of curiosity, do you know if that happens frequently or did it occur during GitHub outages?

Reply:
If I create a dummy book and include the link, then it's fine. But these links are verified in a batch with other GH links (in parallel) and maybe we hit some kind of DoS protection mechanism. Who knows :)

Reply:
> @marxin out of curiosity, do you know if that happens frequently or did it occur during GitHub outages?

In my case, it happens every single time. Similarly, if you take a look at the nightly CI:
https://github.com/rust-lang/rustc-dev-guide/actions/runs/11653521916/job/32445996122

Reply:
I suspect they're actually just rate limits, but yeah anyway, thanks for the info.

Reply:
The rate limits are properly reflected in the return code: `Server returned 429 Too Many Requests for https://github.com`, but these 500s are somehow different.